### PR TITLE
Add Early Link To Intro Doc

### LINKED ISSUE #106600: Update README.md to include examples
### 📚 The doc issue

Update README.md to include impressive short examples of what PyTorch can do for newcomers.

The existing documentation for PyTorch is very well crafted and thorough. However, as it is now one of the most popular machine learning libraries for Python, and more people are coming to machine learning now than ever before, I believe that the beginning of their ReadMe could cater more toward a beginner to machine learning than it does currently. Currently their ReadMe seems more to cater towards those experienced with Machine Learning and why they should use PyTorch, with no examples given within the first several pages. This change adds brief and impressive code snippets to demonstrate how easily one can create power machine learning tools with small amounts of code utilizing the PyTorch library.

### Suggest a potential alternative/fix

Early in the README.md could be updated  with simple and powerful examples that make it clear to newer developers to the project what developers can do with PyTorch.

See example below starting at **line 9** in README.md

You can reuse your favorite Python packages such as NumPy, SciPy, and Cython to extend PyTorch when needed.

## What Can PyTorch Do?
- Define a tensor filled with zeros:
    - ```
      import torch
      zeros_tensor = torch.zeros((3, 3))
      ```
- Compute the element-wise exponential of a tensor:
    - ```
      import torch
      tensor = torch.tensor([1.0, 2.0, 3.0])
      exp_tensor = torch.exp(tensor)
      ```
- Perform a matrix multiplication using PyTorch:
    - ```
      import torch
      A = torch.tensor([[1, 2], [3, 4]])
      B = torch.tensor([[5, 6], [7, 8]])
      result = torch.matmul(A, B)
      ```
- Create a simple neural network layer with PyTorch:
    - ```
      import torch.nn as nn
      layer = nn.Linear(in_features=10, out_features=5)
      ```
Our trunk health (Continuous Integration signals) can be found at [hud.pytorch.org](https://hud.pytorch.org/ci/pytorch/pytorch/main).

cc @svekars @carljparker

### PR DESCRIPTION
**Pull request by Tincaniam**
Fixes #106600 by including an early link to the intro docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Our trunk health (Continuous Integration signals) can be found at [hud.pytorch.o
 
 ## More About PyTorch
 
+[PyTorch Introduction](https://pytorch.org/tutorials/beginner/basics/intro.html)
```
Comment (soulitzer) [inline README.md:47]:
```suggestion
[Learn the basics of PyTorch](https://pytorch.org/tutorials/beginner/basics/intro.html)
```

**Review by soulitzer — COMMENTED** [review]
Thanks for the addition, I made a small suggestion on the link name

**Comment by Tincaniam** [comment]
Looks great, thanks!

**Review by albanD — APPROVED** [review]
Thanks!

**Review by soulitzer — APPROVED** [review]

**Pull request closed without merging**